src/modules/manga-details/components/MangaDetails.tsx

```diff
@@ -0,0 +1,65 @@
+import { useQuery } from '@tanstack/react-query';
+import { ISelectedManga, ISelectedChapter } from '../../common/types/app-types';
+import { fetchChapterListInfo, fetchComicInfo } from '../../common/services/api-service';
+import useMangaStore from '../../common/stores/store';
+import { chapterListResponseFilter, comicResponseFilter } from '../../common/helpers/response-filters';
+import ChapterList from './ChapterList';
+import { useParams } from 'react-router';
+
+
+const MangaDetails = () => {
+    const { selectedSlug } = useParams()!
+    const { isPending: isPendingComic, error: errorComic, data: ComicInfo } = useQuery({
+        queryKey: ['fetchComicInfo', selectedSlug],
+        queryFn: () => fetchComicInfo(selectedSlug!),
+        enabled: !!selectedSlug
```

Comment:
Should ```ComicInfo``` not also be camelCase like ```isPendingComic``` and ```errorComic```?

Reply:
Oh, yes you are right. Thanks for the heads up!